MapboxCoreNavigation/EventsManager.swift

```diff
@@ -275,21 +312,17 @@ extension EventsManager {
     
     //MARK: - Session State Management
     @objc private func didChangeOrientation(_ notification: NSNotification) {
-        if sessionState != nil {
-            sessionState.reportChange(to: UIDevice.current.orientation)
-        }
+        sessionState?.reportChange(to: UIDevice.current.orientation)
     }
     
     @objc private func didChangeApplicationState(_ notification: NSNotification) {
-        if sessionState != nil {
-            sessionState.reportChange(to: UIApplication.shared.applicationState)
-        }
+        sessionState?.reportChange(to: UIApplication.shared.applicationState)
     }
     
     @objc private func applicationWillTerminate(_ notification: NSNotification) {
-        if !sessionState.terminated {
+        if let session_state = sessionState, !session_state.terminated {
```

Comment:
This needs some stylistic cleanup. Might as well use the local non-optional below if we can acquire it.